test/api/league_test.py

```diff
@@ -0,0 +1,147 @@
+                                                     return_value={'username': 'nepnep'})
+        self.create_test_user()
+
+    def test_get_returns_all_leagues_associated_to_user(self):
+        league_n = create_league(self.user, name="Nep League", rating_scheme='ELO')
+        league_m = create_league(self.user, name="Noire League", rating_scheme='type1')
+
+        data = json.loads(self.app.get('/api/league/').data)
+
+        expected_n = {
+            'league_id': league_n.league_id,
+            'rating_scheme': league_n.rating_scheme,
+            'name': league_n.name,
+            'description': league_n.description,
+            'participant_count': league_n.participant_count
+        }
+        expected_m = {
+            'league_id': league_m.league_id,
+            'rating_scheme': league_m.rating_scheme,
+            'name': league_m.name,
+            'description': league_m.description,
+            'participant_count': league_m.participant_count
+        }
+
+        self.assertTrue(expected_n in data and
+                        expected_m in data)
+
+    def test_get_does_not_return_leagues_not_associated_to_user(self):
+        self.create_test_user()
+        different_user = create_user('gege', name='Nepgear', company_name='Planeptune')
+
+        league_n = create_league(different_user, name="Nep League", rating_scheme='ELO')
+        league_m = create_league(different_user, name="Noire League", rating_scheme='ELO')
+
+        data = json.loads(self.app.get('/api/league/').data)
+
+        expected_n = {
+            'league_id': league_n.league_id,
+            'rating_scheme': league_n.rating_scheme,
+            'name': league_n.name,
+            'description': league_n.description,
+            'participant_count': league_n.participant_count
+        }
+        expected_m = {
+            'league_id': league_m.league_id,
+            'rating_scheme': league_m.rating_scheme,
+            'name': league_m.name,
+            'description': league_m.description,
+            'participant_count': league_m.participant_count
+        }
+        self.assertFalse(expected_n in data or
+                         expected_m in data)
+
+    def test_get_returns_empty_list_if_no_leagues_are_associated_to_user(self):
+        response = self.app.get('/api/league/')
+        self.assertEqual(json.loads(response.data), [])
+
+    def test_post_returns_400_if_invalid_rating_scheme_is_specified(self):
+        self.args_mock.return_value = {
+            'username': 'nepnep',
+            'rating_scheme': 'wrong',
+        }
+        response = self.app.post('/api/league/')
+        self.assertEqual(response.status_code, 400)
+
+    def test_post_new_league_when_created_successfully(self):
+        self.args_mock.return_value = {
+            'username': 'nepnep',
+            'rating_scheme': 'type1',
+        }
+        response = self.app.post('/api/league/')
+        self.assertEqual(response.status_code, 200)
```

Comment:
assert the league returned in the response has the dictionary structre you expect, like you did in ``` test_get_returns_all_leagues_associated_to_user ```